Aquifer/IO.swift

```diff
@@ -12,13 +12,13 @@ import Foundation
 import Swiftz
 
 /// Returns a `Pipe` that reads input from `stdin` line-by-line and terminates on end-of-input.
-public func stdinLn<UO, UI>() -> Proxy<UO, UI, (), String, ()> {
+public func stdinLn() -> Producer<String, ()>.T {
```

Comment:
Actually, I think I may have got it working now.  Needed a few modifications of types in other places (description and debugDescription), but it works.